scripts/sct_create_mask.py

```diff
@@ -299,15 +296,13 @@ def create_line(param, fname, coord, nz):
     # set all voxels to zero
     sct.run(['sct_maths', '-i', 'line.nii', '-mul', '0', '-o', 'line.nii'], param.verbose)
 
-    create_add_str = ""
-    for iz in range(nz):
-        if iz == nz - 1:
-            create_add_str += str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1'
-        else:
-            create_add_str += str(int(coord[0])) + ',' + str(int(coord[1])) + ',' + str(iz) + ',1:'
+    if isinstance(coord[0], Coordinate):
```

Comment:
i've noticed a change of behaviour, and i _think_ it comes from there. Below is a result from the following syntax:
~~~
sct_create_mask -i t2.nii.gz -p point,label.nii -o mask_point.nii
~~~
The gif anim alternates between master (c3054eb25ab9befc920446be8282bac05ccfb2a4) and aj/2870-refactor_sct_label_utils (50daa2e24575e8026982de14488d500e5ba73c71):
![anim](https://user-images.githubusercontent.com/2482071/94577473-a4274200-0244-11eb-83d6-392fe335b7af.gif)

As can be seen, the mask generated from the code in this branch only spans a single slice, while it should extend across all the slices in the superior-inferior direction. The "create_line" function is supposed to create a vertical line out of the input single coord, and i think that when inputing a single coord (Coordinate) object, it does not iterate across z (while it should).

Reply:
wow good catch. Investigating.

Reply:
@jcohenadad could you share the exact input files you used to reproduce the issue? That would make my life easier.

Reply:
github is buggy... cannot upload. So here you go: https://www.dropbox.com/s/klgfekh16k4bry2/Archive.zip?dl=0

Reply:
Fixed in 0a6ca12

Reply:
tested with:
- <coord,XxY>
- <point,FILE>
- <center>
- <centerline,FILE>

all good!